R/generate_dust_sexp.R

```diff
@@ -10,12 +10,23 @@ generate_dust_sexp <- function(expr, dat, options = list()) {
       ret <- sprintf("-%s", args[[1]])
     } else if (n == 1 && fn == "+") {
       ret <- args[[1]]
-    } else if (n == 2 && fn %in% c("+", "-", "*", "/")) {
+    } else if (n == 2 && fn %in% c("+", "-", "*", "/", "==")) {
```

Comment:
Just to check, are we happy here with `==` re the usual floating point comparison story?